sdk/clientcore/core/src/test/java/io/clientcore/core/http/RestProxyTests.java

```diff
@@ -215,4 +227,29 @@ public void doesNotChangeEncodedPath() throws IOException {
 
         testInterface.testListNext(nextLinkUri).close();
     }
+
+    @Test
```

Comment:
Added this test in RestProxy for confirming e2e scenario alongside codegen testing.